proposals/repository.go

```diff
@@ -72,16 +75,16 @@ func (r *Repository) fetch(ctx context.Context, status string) ([]*models.GetPro
 	return resp.GetPayload().Proposals, nil
 }
 
-func (r *Repository) GetVotes(ctx context.Context, proposalID int) ([]*models.GetProposalVotesResultVotes, error) {
-	page := 1.0
-	votes := make([]*models.GetProposalVotesResultVotes, 0)
+func (r *Repository) GetVotes(ctx context.Context, proposalID int) ([]*query.VotesOKBodyVotesItems0, error) {
```

Comment:
I've played around fcd fetching votes endpoints and I see that there doesn't seem to be an endpoint for getting complete votes list for finished proposals:
- `https://fcd.terra.dev/cosmos/gov/v1beta1/proposals/{proposal_id}/votes` works only for voting proposals, for the finished and deposited ones it simply returns nothingness;
- `https://fcd.terra.dev/v1/gov/proposals/{proposal_id}/votes` works up to the very beginning, but returns a lesser amount of votes.

I assumed maybe the first endpoint expands the validator's unvoted delegators votes, but it doesn't.

That sucks, and also that deceives the user of the method now. What do you think of making the name more specific? e.g. GetVotingProposalVotes. Or maybe an explanation comment would be better